Deprecate ger ger.out and bump version from 9 to 10

cccclai opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #72192
* #72191



Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636)

Commit:
Deprecate ger ger.out and bump version from 9 to 10

Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

[ghstack-poisoned]

Commit:
Update on "Deprecate ger ger.out and bump version from 9 to 10"


Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

[ghstack-poisoned]

Commit:
Update on "Deprecate ger ger.out and bump version from 9 to 10"


Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

[ghstack-poisoned]

Commit:
Update on "Deprecate ger ger.out and bump version from 9 to 10"


Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

[ghstack-poisoned]

Commit:
Update on "Deprecate ger ger.out and bump version from 9 to 10"




Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636)

[ghstack-poisoned]

Commit:
Update on "Deprecate ger ger.out and bump version from 9 to 10"




Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636)

[ghstack-poisoned]

mruberry commented:
hey @cccclai, thanks for pursuing this deprecation (cc @Lezcano)! Would you ping me when the test failures are resolved? Also not I think you'll need to remove `ger`'s rst entries for the docs, too.

cccclai commented:
> hey @cccclai, thanks for pursuing this deprecation (cc @Lezcano)! Would you ping me when the test failures are resolved? Also not I think you'll need to remove `ger`'s rst entries for the docs, too.

Thanks for checking! I'm waiting for the change https://github.com/pytorch/pytorch/issues/39853 and will rebase/fix test after it is merged.

Commit:
Update on "Deprecate ger ger.out and bump version from 9 to 10"




Deprecate aten::ger and aten::ger.out and bump version from 9 to 10

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636/)

Differential Revision: [D33910636](https://our.internmc.facebook.com/intern/diff/D33910636)

[ghstack-poisoned]
